ITensorGaussianMPS/src/gmps.jl

```diff
@@ -165,6 +265,33 @@ end
 #
 # Correlation matrix diagonalization
 #
+abstract type AbstractSymmetry end
+
+struct ConservingNfParity{T} <: AbstractSymmetry
```

Comment:
I would vote for `ConservesNf` and `ConservesNfParity` instead of `Conserving*`, I think that makes it sound more like a property of the object and also is closer to how we might actually say it (i.e. "the state conserves particle number" vs. "the state is particle number conserving").

Reply:
Ok.